{
	"affectedProducts": [
		"Rocky Linux 8"
	],
	"cves": [
		{
			"cvss3BaseScore": "9.8",
			"cvss3ScoringVector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H",
			"cwe": "CWE-190",
			"name": "CVE-2019-25032",
			"sourceBy": "MITRE",
			"sourceLink": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-25032"
		},
		{
			"cvss3BaseScore": "9.8",
			"cvss3ScoringVector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H",
			"cwe": "CWE-190->CWE-787",
			"name": "CVE-2019-25034",
			"sourceBy": "MITRE",
			"sourceLink": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-25034"
		},
		{
			"cvss3BaseScore": "9.8",
			"cvss3ScoringVector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H",
			"cwe": "CWE-787",
			"name": "CVE-2019-25035",
			"sourceBy": "MITRE",
			"sourceLink": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-25035"
		},
		{
			"cvss3BaseScore": "9.8",
			"cvss3ScoringVector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H",
			"cwe": "CWE-617->CWE-787",
			"name": "CVE-2019-25036",
			"sourceBy": "MITRE",
			"sourceLink": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-25036"
		},
		{
			"cvss3BaseScore": "7.5",
			"cvss3ScoringVector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
			"cwe": "CWE-617",
			"name": "CVE-2019-25037",
			"sourceBy": "MITRE",
			"sourceLink": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-25037"
		},
		{
			"cvss3BaseScore": "9.8",
			"cvss3ScoringVector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H",
			"cwe": "CWE-190->CWE-787",
			"name": "CVE-2019-25038",
			"sourceBy": "MITRE",
			"sourceLink": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-25038"
		},
		{
			"cvss3BaseScore": "9.8",
			"cvss3ScoringVector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H",
			"cwe": "CWE-190->CWE-787",
			"name": "CVE-2019-25039",
			"sourceBy": "MITRE",
			"sourceLink": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-25039"
		},
		{
			"cvss3BaseScore": "7.5",
			"cvss3ScoringVector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
			"cwe": "CWE-835",
			"name": "CVE-2019-25040",
			"sourceBy": "MITRE",
			"sourceLink": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-25040"
		},
		{
			"cvss3BaseScore": "7.5",
			"cvss3ScoringVector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
			"cwe": "CWE-617",
			"name": "CVE-2019-25041",
			"sourceBy": "MITRE",
			"sourceLink": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-25041"
		},
		{
			"cvss3BaseScore": "9.8",
			"cvss3ScoringVector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H",
			"cwe": "CWE-787",
			"name": "CVE-2019-25042",
			"sourceBy": "MITRE",
			"sourceLink": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-25042"
		},
		{
			"cvss3BaseScore": "4.4",
			"cvss3ScoringVector": "CVSS:3.1/AV:L/AC:L/PR:H/UI:N/S:U/C:N/I:H/A:N",
			"cwe": "CWE-59",
			"name": "CVE-2020-28935",
			"sourceBy": "MITRE",
			"sourceLink": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-28935"
		}
	],
	"description": "The unbound packages provide a validating, recursive, and caching DNS or DNSSEC resolver. \n\nSecurity Fix(es):\n\n* unbound: integer overflow in the regional allocator via regional_alloc (CVE-2019-25032)\n\n* unbound: integer overflow in sldns_str2wire_dname_buf_origin can lead to an out-of-bounds write (CVE-2019-25034)\n\n* unbound: out-of-bounds write in sldns_bget_token_par (CVE-2019-25035)\n\n* unbound: assertion failure and denial of service in synth_cname (CVE-2019-25036)\n\n* unbound: assertion failure and denial of service in dname_pkt_copy via an invalid packet (CVE-2019-25037)\n\n* unbound: integer overflow in a size calculation in dnscrypt/dnscrypt.c (CVE-2019-25038)\n\n* unbound: integer overflow in a size calculation in respip/respip.c (CVE-2019-25039)\n\n* unbound: infinite loop via a compressed name in dname_pkt_copy (CVE-2019-25040)\n\n* unbound: assertion failure via a compressed name in dname_pkt_copy (CVE-2019-25041)\n\n* unbound: out-of-bounds write via a compressed name in rdata_copy (CVE-2019-25042)\n\n* unbound: symbolic link traversal when writing PID file (CVE-2020-28935)\n\nFor more details about the security issue(s), including the impact, a CVSS score, acknowledgments, and other related information, refer to the CVE page(s) listed in the References section.\n\nAdditional Changes:\n\nFor detailed information on changes in this release, see the Rocky Linux 8.4 Release Notes linked from the References section.",
	"fixes": [
		{
			"sourceBy": "Red Hat",
			"sourceLink": "https://bugzilla.redhat.com/show_bug.cgi?id=1714175",
			"ticket": "1714175"
		},
		{
			"sourceBy": "Red Hat",
			"sourceLink": "https://bugzilla.redhat.com/show_bug.cgi?id=1842837",
			"ticket": "1842837"
		},
		{
			"sourceBy": "Red Hat",
			"sourceLink": "https://bugzilla.redhat.com/show_bug.cgi?id=1850460",
			"ticket": "1850460"
		},
		{
			"sourceBy": "Red Hat",
			"sourceLink": "https://bugzilla.redhat.com/show_bug.cgi?id=1878761",
			"ticket": "1878761"
		},
		{
			"sourceBy": "Red Hat",
			"sourceLink": "https://bugzilla.redhat.com/show_bug.cgi?id=1954772",
			"ticket": "1954772"
		},
		{
			"sourceBy": "Red Hat",
			"sourceLink": "https://bugzilla.redhat.com/show_bug.cgi?id=1954778",
			"ticket": "1954778"
		},
		{
			"sourceBy": "Red Hat",
			"sourceLink": "https://bugzilla.redhat.com/show_bug.cgi?id=1954780",
			"ticket": "1954780"
		},
		{
			"sourceBy": "Red Hat",
			"sourceLink": "https://bugzilla.redhat.com/show_bug.cgi?id=1954782",
			"ticket": "1954782"
		},
		{
			"sourceBy": "Red Hat",
			"sourceLink": "https://bugzilla.redhat.com/show_bug.cgi?id=1954794",
			"ticket": "1954794"
		},
		{
			"sourceBy": "Red Hat",
			"sourceLink": "https://bugzilla.redhat.com/show_bug.cgi?id=1954796",
			"ticket": "1954796"
		},
		{
			"sourceBy": "Red Hat",
			"sourceLink": "https://bugzilla.redhat.com/show_bug.cgi?id=1954797",
			"ticket": "1954797"
		},
		{
			"sourceBy": "Red Hat",
			"sourceLink": "https://bugzilla.redhat.com/show_bug.cgi?id=1954799",
			"ticket": "1954799"
		},
		{
			"sourceBy": "Red Hat",
			"sourceLink": "https://bugzilla.redhat.com/show_bug.cgi?id=1954801",
			"ticket": "1954801"
		},
		{
			"sourceBy": "Red Hat",
			"sourceLink": "https://bugzilla.redhat.com/show_bug.cgi?id=1954804",
			"ticket": "1954804"
		}
	],
	"name": "RLSA-2021:1853",
	"publishedAt": "2021-05-18T06:15:09Z",
	"rpms": {
		"Rocky Linux 8": {
			"nvras": [
				"python3-unbound-0:1.7.3-15.el8.aarch64.rpm",
				"python3-unbound-0:1.7.3-15.el8.x86_64.rpm",
				"python3-unbound-debuginfo-0:1.7.3-15.el8.aarch64.rpm",
				"python3-unbound-debuginfo-0:1.7.3-15.el8.x86_64.rpm",
				"unbound-0:1.7.3-15.el8.aarch64.rpm",
				"unbound-0:1.7.3-15.el8.src.rpm",
				"unbound-0:1.7.3-15.el8.x86_64.rpm",
				"unbound-debuginfo-0:1.7.3-15.el8.aarch64.rpm",
				"unbound-debuginfo-0:1.7.3-15.el8.i686.rpm",
				"unbound-debuginfo-0:1.7.3-15.el8.x86_64.rpm",
				"unbound-debugsource-0:1.7.3-15.el8.aarch64.rpm",
				"unbound-debugsource-0:1.7.3-15.el8.i686.rpm",
				"unbound-debugsource-0:1.7.3-15.el8.x86_64.rpm",
				"unbound-devel-0:1.7.3-15.el8.aarch64.rpm",
				"unbound-devel-0:1.7.3-15.el8.i686.rpm",
				"unbound-devel-0:1.7.3-15.el8.x86_64.rpm",
				"unbound-libs-0:1.7.3-15.el8.aarch64.rpm",
				"unbound-libs-0:1.7.3-15.el8.i686.rpm",
				"unbound-libs-0:1.7.3-15.el8.x86_64.rpm",
				"unbound-libs-debuginfo-0:1.7.3-15.el8.aarch64.rpm",
				"unbound-libs-debuginfo-0:1.7.3-15.el8.i686.rpm",
				"unbound-libs-debuginfo-0:1.7.3-15.el8.x86_64.rpm"
			]
		}
	},
	"severity": "SEVERITY_MODERATE",
	"shortCode": "RL",
	"synopsis": "Moderate: unbound security, bug fix, and enhancement update",
	"topic": "An update is available for unbound.\nThis update affects Rocky Linux 8.\nA Common Vulnerability Scoring System (CVSS) base score, which gives a detailed severity rating, is available for each vulnerability from the CVE list",
	"type": "TYPE_SECURITY"
}
